scripts/ci/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -ex
+
+ROOTDIR="$(dirname "$0")/../.."
+cd "${ROOTDIR}"
+
+# shellcheck disable=SC2166
+if [ "$CIRCLE_BRANCH" = release -o -n "$CIRCLE_TAG" -o -n "$FORCE_RELEASE" ]; then echo -n >prerelease.txt; else date -u +"nightly.%Y.%-m.%-d" >prerelease.txt; fi
+if [ -n "$CIRCLE_SHA1" ]
+then
+  echo -n "$CIRCLE_SHA1" >commit_hash.txt
+fi
+
+mkdir -p build
+cd build
+
+# shellcheck disable=SC2166
+[ -n "$COVERAGE" -a "$CIRCLE_BRANCH" != release -a -z "$CIRCLE_TAG" ] && CMAKE_OPTIONS="$CMAKE_OPTIONS -DCOVERAGE=ON"
+
+# shellcheck disable=SC2086
+cmake .. -DCMAKE_BUILD_TYPE="${CMAKE_BUILD_TYPE:-Release}" $CMAKE_OPTIONS -G "Unix Makefiles"
+
+[[ "${MAKE_JOBS}" != "" ]] || MAKE_JOBS=4
+make -j "${MAKE_JOBS}"
```

Comment:
Hm... I guess this *one* would be fine... but we *really* should not change this in this PR and keep it the same as the circleci config...

Reply:
By the way: the canonical way to do this configurably would be to set ``MAKEFLAGS`` to ``-j 4`` instead...

Reply:
I suggested that, but yeah doing it in a separate PR sounds fine too.

Reply:
Ok, removed that.